pkg/network/node/ovs/ovs.go

```diff
@@ -162,7 +162,7 @@ func (ovsif *ovsExec) execWithStdin(cmd string, stdinArgs []string, args ...stri
 
 	output, err := kcmd.CombinedOutput()
 	if err != nil {
-		klog.V(2).Infof("Error executing %s: %s", cmd, string(output))
+		klog.Errorf("Error executing cmd: %s with args: %v, output: %s", cmd, args, string(output))
```

Comment:
> but I think maybe there aren't any such cases in ovs.go?

Yes,[ certain places accept an error,](https://github.com/openshift/sdn/blob/master/pkg/network/node/ovs/ovs.go#L184) but never expect an error. 

> do output:\n%s since output itself may have newlines in it so it's probably clearer to start it on a new line?

OK

> Also, the ovs logging changes should be a separate commit (not necessarily a separate PR) from the healthcheck changes

OK